packages/core/src/fil/depositAddress.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  FilMasterWalletData,
+  FilWallet,
+  FilWalletData,
+} from "./abstractWallet";
+import { Client } from "../httpClient";
+import { Balance, Key, Keychains } from "../types";
+import {
+  BalanceDTO,
+  DepositAddressDTO,
+  PatchWalletNameRequest,
+} from "../__generate__/fil";
+import { BNConverter } from "../utils/common";
+
+export interface FilDepositAddressData
+  extends Omit<FilWalletData, "encryptionKey"> {}
```

Comment:
Why omit this? It's not in FilWalletData

Reply:
But WalletData has encryptionKey, so i omitted it.

```
export interface WalletData {
  id: string;
  name: string;
  address: string;
  encryptionKey: string;
  createdAt: string;
  updatedAt: string;
  status: WalletStatus;
  version?: string;
}
```

Reply:
Ah ok!